wg-charter/01-01

```diff
@@ -39,35 +31,29 @@ different applications can adapt to their own needs.
 
 While authentication is a key goal of this working group, it is not
 the objective of this working group to develop new authentication
-technologies.  Rather, the security protocol developed by this
-group will provide a way to leverage existing authentication
-technologies to associate identities with keys used in the protocol,
-just as TLS does with X.509.
+technologies.  Rather, the MLS protocol provides a way to leverage
+existing authentication technologies to associate identities with
+keys used in the protocol, just as TLS does with X.509.
 
-In developing this protocol, we will draw on lessons learned from
-several prior message-oriented security protocols, in addition to
-the proprietary messaging security protocols deployed within
+While developing the MLS protocol, the group drew on lessons learned
+from several prior message-oriented security protocols, in addition
+to the proprietary messaging security protocols deployed within
 existing applications:
 
 o S/MIME - https://tools.ietf.org/html/rfc5751
 o OpenPGP - https://tools.ietf.org/html/rfc4880
 o Off the Record - https://otr.cypherpunks.ca/Protocol-v3-4.1.1.html
 o Double Ratchet - https://en.wikipedia.org/wiki/Double_Ratchet_Algorithm
 
-The intent of this working group is to follow the pattern of
-TLS 1.3, with specification, implementation, and verification
-proceeding in parallel.  By the time we arrive at RFC, we
-hope to have several interoperable implementations as well
-as a thorough security analysis.
-
-The specifications developed by this working group will be
-based on pre-standardization implementation and deployment
-experience, and generalizing the design described in:
-
-o draft-omara-mls-architecture
-o draft-barnes-mls-protocol
+The working group followed the pattern of TLS 1.3, with specification,
+implementation, and verification proceeding in parallel.  When we arrived
+at RFC, we had several interoperable implementations as well as a thorough
+security analysis.
 
 Note that consensus is required both for changes to the protocol mechanisms
 from these documents and retention of the mechanisms from them. In particular,
 because something is in the initial document set does not imply that there is
 consensus around the feature or around how it is specified.
+
+Now that MLS has been published the group will work on MLS protocol extensions
+and cipher suites.
```

Comment:
Overall, I think these changes are good.  I agree with @ekr's comments at IETF 118 that we should have some scoping around what extensions we will develop.  I surveyed the existing proposals, and I think they point to a few buckets:

```suggestion
Now that MLS has been published the group will work on the following MLS protocol extensions:

* Support for use of MLS in protocols developed by the MIMI working group
* Support for Verifiable Credentials
* Support for common operational patterns in messaging applications
* Quantum-safe cipher suites
* Framework for safe extensibility
* Detection of lost application messages
* Support for sending messages to individual members of a group
```

This taxonomy is based on the current document set:

* Support for use of MLS in protocols developed by the MIMI working group
    * SelfRemove proposal (draft-mahy-mlm-selfremove)
    * Room Policy (draft-mahy-mls-room-policy-ext)
    * Content Advertisement (draft-ietf-mls-extensions)
* Support for Verifiable Credentials
    * UserInfo VC (draft-barnes-mls-addl-creds)
    * Multi-Cred (draft-barnes-mls-addl-creds)
    * Weak Multi-Cred (draft-barnes-mls-addl-creds)
* Support for common operational patterns in messaging applications
    * KeyPackage context (draft-mahy-mls-kp-context)
    * Last resort KeyPackages (draft-ietf-mls-extensions)
* Quantum-safe ciphersuites
    * PQ-Hybrid KEM (draft-mahy-mls-x25519kyber768draft00)
* Framework for safe extensibility
    * Safe Extensions (draft-ietf-mls-extensions)
* Detection of lost application messages
    * AppAck (draft-ietf-mls-extensions)
* Support for sending messages to individual members of a group
    * Targeted Messages (draft-ietf-mls-extensions)

Reply:
I think this is a good list, but I feel that SelfRemove and Content Advertisement is part of "a common operational pattern in messaging"